src/corona_hakab_model/common/util.py

```diff
@@ -13,6 +14,10 @@ def uniform_dist(a, b):
         range_to_choose_from = list(range(a, b+1))
         return partial(lambda size=None: np.random.choice(range_to_choose_from, size=size))
 
+    def weighted_dist(elements, p):
+        return partial(lambda size=None: random.choices(elements, weights=p, k=size))
```

Comment:
Why not use numpy? No need for new module...

Reply:
changed to np.random.choice in lastest commit. 
Also, it seems that there's a rounding problem in many of the given days' distributions in the excel sheet. So in such cases (i.e. sum of probs. isn't 1), I just normalize the list of probabilities.